Formula/organize-tool.rb

```diff
@@ -0,0 +1,171 @@
+class OrganizeTool < Formula
+  include Language::Python::Virtualenv
+
+  desc "Organize - The file management automation tool"
```

Comment:
```suggestion
  desc "File management automation tool"
```

Descriptions shouldn't begin with the formula's name. While the name is `organize-tool` here (which is why the audit didn't catch this), we would want to avoid having a part of the formula/program's name in the description.

When `brew desc organise-tool` is run:
```
organize-tool: File management automation tool
```
is preferable to what what would happen currently:
```
organize-tool: Organize - The file management automation tool
```